### DISCUSSION TITLE
Using Ethereum Auth0 instead of Ethreum web3 auth

### DISCUSSION THREAD

**Discussion by lambdina** [Web3Auth/Web3Auth#1012] [Help]
Hello,
I wanted to be able to use all of the features from auth0 (the one that is interesting to me is Hasura) So I linked my web3auth to auth0.
With Auth0, I enabled facebook, Google, and Siwe (Ethereum).
However, It's not working. 
The code I am using is literally this one : https://github.com/Web3Auth/examples/tree/main/web-core-sdk/custom-authentication/auth0-react-core-example

The error message is : 

> There seems to be some bug in the code. Please contact support to fix this.
> Cannot%20read%20properties%20of%20undefined%20(reading%20'get')

I am using Chromium.

Before we added Ethereum, it was working with Just facebook and Google enabled on Auth0 side. Now, even when we tried to disable Ethereum on Auth0, it still does not works.

Thanks.

**Comment by kmchmk** [comment]
I don't think that this is related to the Eth wallet. I think that there is some server-side issue.

**Reply by kmchmk** [reply]
It is working now.
